tests/core/pyspec/eth2spec/test/electra/sanity/test_slots.py

```diff
@@ -0,0 +1,125 @@
+from eth2spec.test.context import (
+    spec_state_test,
+    with_electra_and_later,
+)
+from eth2spec.test.helpers.deposits import prepare_pending_deposit
+from eth2spec.test.helpers.state import transition_to
+
+
+def run_epoch_processing(spec, state, pending_deposits=[], pending_consolidations=[]):
```

Comment:
nitpicking: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html#best-practice

```suggestion
def run_epoch_processing(spec, state, pending_deposits=None, pending_consolidations=None):
    if pending_deposits is None:
        pending_deposits = []
    if pending_consolidations is None:
        pending_consolidations = []
```